data-model/src/main/java/org/alfresco/service/cmr/search/SearchParameters.java

```diff
@@ -201,6 +201,8 @@ public enum Operator
 
     private String timezone;
     
+    private int trackTotalHits = 0;
```

Comment:
Please could we have a javadoc comment somewhere explaining what this field holds?

Reply:
From PMD:
> SearchParameters.java:204:	RedundantFieldInitializer:	Avoid using redundant field initializer for 'trackTotalHits'